src/foam/nanos/console/Console.js

```diff
@@ -413,12 +429,15 @@ YYYY-MM-DDTHH:MM
     async function eval_(cmd) {
       var self = this;
       cmd = cmd.trim();
+      this.clearProperty('historyPosition');
       if ( ! cmd ) return;
       this.addHistory(cmd);
       this.outputDiv.tag('br').start().show(self.showPrompts$).start('b').add('> ').end().add(cmd);
 
+      // with is deprecated
```

Comment:
Deprecated but never to be removed. There's no other way to do this kind of thing and it's exactly what the Chrome console code does.